conf/plants/mlss_conf.xml

```diff
@@ -635,6 +635,7 @@
     <protein_trees
       collection="oat_cultivars"
       homology_range_index="34"
+      prefer_for_genomes="avena_sativa_ot3098"
```

Comment:
The `prefer_for_genomes` attribute is unnecessary in this case because `avena_sativa_ot3098` is only in one cultivar protein-tree collection.
```suggestion
```